05_syntax_lab/02.py

```diff
@@ -1,6 +1,7 @@
-"""
-Write a Python program that randomizes 7 numbers
-and prints their sum total.
-If the sum is divisable by 7, also print the word "Boom"
-"""
-
+from random import randint
+
+sum = int(0)

```

Comment:
Notice how github colours "sum" in the same color as print (and differently from "number" and "Boom"). The reason is python has a function named sum, so when you use that as a variable name it can be a bit confusing.